app/assets/stylesheets/styles/components/guides/components/_step_1.css.scss

```diff
@@ -27,4 +27,12 @@
       color: #FF3366;
     }
   }
+
+  .crop-image {
+    width: 100%;
+    height: 200px;
+    background-color: #F5F5F5;
```

Comment:
Color literals like `#F5F5F5` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#F5F5F5` should be written as `#f5f5f5`